nuget.config

```diff
@@ -1,15 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <config>
-    <clear />
     <add key="repositoryPath" value="$\..\packages" />
   </config>
   <disabledPackageSources>
     <clear />
   </disabledPackageSources>
   <packageSources>
     <!-- For more info, see https://docs.nuget.org/consume/nuget-config-file -->
-    <clear />

```

Comment:
><clear /> [](http://example.com/codeflow?start=4&length=9)

Is this necessary? To the best of my knowledge WinUI Gallery doesn't depend on any of our internal feeds, and when this is being used as a submodule it's too deep in the file directory to influence the WinUI build.

Reply:
this is necessary to allow a single feed configuration (best practice) so WinUI Gallery picks up its nuget.org dependencies indirectly via the single internal WinUI feed upstream.

Reply:
According to the **NuGet and MyGet multi-feed detection** page of the site you linked, this `nuget.config` would fail the check. (There's actually no way it can pass the check whether or not the `<clear />` is present.) Can we clobber/delete this file when initing the submodule so the root WinUI `nuget.config` is used?

Reply:
this nuget.config is currently being built without error.  we can experiment with removing it, but I made the change to enable the build.